ci/ray_ci/ray_docker_container.py

```diff
@@ -56,9 +56,15 @@ def run(self) -> None:
         self.run_script(cmds)
 
     def _should_upload(self) -> bool:
+        if not self.upload:
+            return False
+        if os.environ.get("BUILDKITE_PIPELINE_ID") != POSTMERGE_PIPELINE:
+            return False
+        if os.environ.get("BUILDKITE_BRANCH").startswith("releases/"):
```

Comment:
one last thing, could you add a default value of `""` (empty string) on the `environ.get()`? otherwise, it can panic here (returning `None`) when `BUILDKITE_BRANCH` is not set.

Reply:
actually, let me just merge this now. and could you make that change in a follow up?